.travis.yml

```diff
@@ -12,14 +12,12 @@ env:
         - SYMFONY_PHPUNIT_DIR=$HOME/.phpunit-bridge
         - SYMFONY_PHPUNIT_REMOVE='' # don't remove prophecy
 
-php: [7.2, 7.1, 7.0, 5.6, 5.5, 5.4]
+php: [7.2, 7.1]
```

Comment:
Dropping support for older PHP version in a PR described as bumping Symfony version ? That's not good description of what the PR does.

Reply:
You're right but without those drops the CI won't go green.
After this https://github.com/Incenteev/ParameterHandler/pull/129 is merged, this change won't be there anymore.